C++/Snippets/fora.sublime-snippet

```diff
@@ -0,0 +1,13 @@
+<snippet>
+    <content><![CDATA[
+for(auto ${2:i} = ${1}.begin(); ${2:i} != ${1}.end(); ${2:i}++)
```

Comment:
IMO, as in https://github.com/sublimehq/Packages/blob/master/C%2B%2B/Snippets/forv.sublime-snippet,
- No new line after `<![CDATA[`.
- A space between `for` and `(`.